brainvisa/toolboxes/data management/processes/database_qc_table.py

```diff
@@ -24,7 +24,8 @@
     'data_filters', ListOf(String()),
     'keys', ListOf(String()),
     'type_labels', ListOf(String()),
-    'output_file', WriteDiskItem('Text File', 'HTML PDF'),
+    'output_file', WriteDiskItem(
```

Comment:
How the format 'HTML PDF' was working, or I did not find were it was declare may be ?

Reply:
It'sa "formats list" defined in brainvisa.tools.aimsGlobals which contains the HTML and optionally the PDF format only if the "wkhtmltopdf" program is found (html top pdf converter).
I suppose we can use `['HTML PDF', 'CSV file']`

Reply:
Ah yes like 'aims writeable volume formats'. I tried `['HTML PDF', 'CSV file']` but it does not work, it say :
`'HTML PDF' n'est pas un format valide`.

Reply:
Ah crap, it's not recursive.
Then try `getFormats('HTML PDF') + ['CSV file']`